mitodl/ocw-studio
# Pt/read title from pdf metadata

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/1495.

#### What's this PR do?
If a PDF is uploaded to Google Drive and synced, and if the PDF contains `Title` in its metadata, the update in this PR populates the `Title` field in OCW Studio. If an existing resource has its title changed (and replaced in Google Drive), then its `Title` field will be updated.

#### How should this be manually tested?
Upload a test PDF via Google Drive and sync to Studio. The `Title` in metadata can be set via Adobe Acrobat or via the `exiftool` command line tool (available via most package managers depending on OS, including `apt` and `homebrew`). For example, 
```
exiftool -Title="TestTitle" mypdf.pdf
``` 
Then, change the `Title` field to something else, for example,
```
exiftool -Title="NewTestTitle" mypdf.pdf
``` 
replace the resource in Google Drive, sync again, and check that the resource's `Title` has been updated in OCW Studio.
